SPECS/samba.spec

```diff
@@ -3324,6 +3299,23 @@ rm -rf %{buildroot}
 %endif # with_clustering_support
 
 %changelog
+* Mon Aug 4 2025 Gaëtan Lehmann <gaetan.lehmann@vates.tech> - 4.10.16-25.2
+- *** Upstream changelog ***
```

Comment:
Need a line for the XCP-ng side of packaging. Like:

```
- Apply packaging changes from XenServer's samba-4.10.16-17.0.{4-8}
```

A comment in the merge commit description would be nice too, saying the above plus the fact that we keep our CentOS7-based samba sources and spec file despite the merge.